samanthas_telegram_bot/data_structures/helper_classes.py

```diff
@@ -0,0 +1,85 @@
+"""Various dataclasses, NamedTuples and TypedDicts for attribute hinting and easy unpacking.
```

Comment:
I suggest to do module `models` and some modules inside of it (as in django orm model-classes). Because now this naming of module isn't very informative

Reply:
Not all classes in there correspond to models, but you have a point. So as a compromise I renamed it into `models_helper_classes` :)

Reply:
I mean it's just some `data` models, so I'm not sure why not just call it `models`?